src/features/PatientInfo/index.js

```diff
@@ -32,7 +32,7 @@ function getMrn(ids) {
   if (ids.length > 0) {
     const mrnTextValues = ["MRN", "MR", "Medical Record Number"]
     const mrn = ids.find(id =>
-      mrnTextValues.includes(id.type?.text.value)||
+      mrnTextValues.includes(id.type?.text?.value)||
       (id.type?.coding && mrnTextValues.includes(id.type?.coding[0].code.value))
```

Comment:
```suggestion
      mrnTextValues.includes(id.type?.text?.value)||
      (id.type?.coding && mrnTextValues.includes(id.type?.coding[0]?.code?.value))
```
`code` might also not exist.